recipe/meta.yaml

```diff
@@ -7,10 +7,10 @@ package:
 source:
     fn: cell_tree2d-{{ version }}-.tar.gz
     url: https://github.com/NOAA-ORR-ERD/cell_tree2d/archive/v{{ version }}.tar.gz
-    sha256: dbdb0f6f3b68536f73b9db79c1199753504fbf58397551b4c9bf210aa364cbce
+    sha256: 4a5759951f67a3ff199137c87bb565b1897541130de5a4a2067acb22b0f39bae
 
 build:
-    number: 0
+    number: 1
```

Comment:
yeah -- would be better to have CI running on the cell_tree2d repo itself.

and all of our other projects :-)

one of these days -- so far, I haven't found the time to figure out how to do it.